### PR TITLE
remove Grafana

### PR DESCRIPTION
**Pull request by halamix2**
**Description**

Changes proposed in this pull request:

- remove Grafana
- remove unused/outdated parts  of documentation

**Related issue(s)**

/kind chore
/area telemetry

### CODE REVIEW

**Review by Sawthis — CHANGES_REQUESTED** [review]
Please remove githubissues from .koapps.yaml and resolve conflicts.

**Review by Sawthis — APPROVED** [review]

**Review by Disper — DISMISSED** [review]

**Review by grego952 — DISMISSED** [review]

File: docs/prow/prow-monitoring.md [hunk 1]
```diff
@@ -123,11 +36,7 @@ The [`sap-kyma-prow-workload`](https://app.google.stackdriver.com/?project=sap-k
  - Long-running GKE clusters (`weekly` and `nightly` clusters)
 
 #### Dashboards
-Stackdriver Monitoring dashboards provide visibility into the performance, uptime, and overall health of long-running Kyma test clusters. Here are the available dashboards:
- - For the [nightly cluster](https://app.google.stackdriver.com/dashboards/2395169590273002360?project=sap-kyma-prow-workloads)
- - For the [weekly cluster](https://app.google.stackdriver.com/dashboards/7169385145780812191?project=sap-kyma-prow-workloads)
-
-Stackdriver Monitoring also provides information about overall [status](https://app.google.stackdriver.com/uptime?project=sap-kyma-prow-workloads) 
+Stackdriver Monitoring provides information about overall [status](https://app.google.stackdriver.com/uptime?project=sap-kyma-prow-workloads)
```
Comment (mmitoraj) [inline docs/prow/prow-monitoring.md:39]:
```suggestion
Stackdriver Monitoring provides information about the overall [status](https://app.google.stackdriver.com/uptime?project=sap-kyma-prow-workloads).
```

**Review by Sawthis — APPROVED** [review]

**Review by Disper — APPROVED** [review]

**Review by Sawthis — APPROVED** [review]

**Review by grego952 — APPROVED** [review]

**Pull request merged**